go/cmd/vttablet/cli/cli.go

```diff
@@ -169,6 +179,36 @@ func run(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func waitForDBAGrants(config *tabletenv.TabletConfig, waitTime time.Duration) error {
+	if waitTime == 0 {
+		return nil
+	}
+	timer := time.NewTimer(waitTime)
+	ticker := time.NewTicker(100 * time.Millisecond)
```

Comment:
@GuptaManan100 and I also discussed the use of this ticker vs `time.Sleep` and for the specific usage here (which is wait at least 100ms before retrying), `Sleep` is an appropriate way to do it.

Reply:
Fixed 👍